Examples/macOS/RandomClips/RandomClips/ViewController.swift

```diff
@@ -104,12 +129,12 @@ class ViewController: NSViewController {
 
                 drumPlayer.currentTime = 0
                 guitarPlayer.currentTime = 0
-
                 drumPlayer.prepare(withFrameCount: 44_100)
                 guitarPlayer.prepare(withFrameCount: 44_100)
 
-                drumLooper.schedule(from: 0, to: drumLooper.duration, avTime: nil)
-                guitarLooper.schedule(from: 0, to: guitarLooper.duration, avTime: nil)
+                drumLooper.play()
```

Comment:
These will not be in sync if started this way.  Should be play(at: futureTime)

Reply:
sorry typo... fixing

Reply:
Ok looks good as far as the example.  I didn't look at the rest.